config/application.yml.example

```diff
@@ -27,8 +27,8 @@ queue_health_check_dead_interval_seconds: '240'
 # How often to enqueue simple jobs to make sure the background queues are running
 queue_health_check_frequency_seconds: '120'
 
-# Must be an even number. 16 seems to be what most sites use.
-recovery_code_length: '16'
+# The number of words in the recovery code phrase
+recovery_code_length: '12'
```

Comment:
I think we can make the number of words fewer if we have more words in the corpus. Add in the Declaration of Independence to the corpus and see whether we can reduce the number of necessary words?